<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Nirmala UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Interviews</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logo/logo.qrc">
    <normaloff>:/logo/logo_icon.png</normaloff>:/logo/logo_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{
background-color: qconicalgradient(cx:0.114, cy:0.113318, angle:134.3, stop:0 rgba(214, 28, 30, 255), stop:1 rgba(255, 255, 255, 255));}</string>
  </property>
  <widget class="QLabel" name="label_APPLICATIONS">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>981</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(214, 30, 32);</string>
   </property>
   <property name="text">
    <string>INTERVIEWS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SEARCH">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>SEARCH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Return_REFERENCE_menu">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>120</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>RETURN PREFERENCE MENU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SUBMITTED_PROJECTS">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>SUBMITTED PROJECTS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-width:3px;
border-radius:20px;

color: rgb(104, 104, 104);
background-color: rgb(255, 255, 255);
border-color: rgb(220, 63, 65);</string>
   </property>
   <property name="placeholderText">
    <string>Enter the text you want to search for here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_EXIT">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>190</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_RECEIVED_PROJECTS">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>RECEIVED PROJECTS</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>861</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>283</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Full Name</string>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Submission Date</string>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Delivery Date</string>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </column>
  </widget>
 </widget>
 <resources>
  <include location="logo/logo.qrc"/>
 </resources>
 <connections/>
</ui>
